.travis.yml

```diff
@@ -15,18 +15,21 @@ rvm:
   - 2.4.5
   - 2.5.5
   - 2.6.2
-  - ruby-head
 jobs:
   include:
     - stage: linting
       rvm: 2.6.2
-      env:
-        - "TEST_TOOL=rubygems"
+      env: "TEST_TOOL=rubygems"
       script: util/ci rubocop
     - stage: test
       rvm: 2.6.2
-      env:
-        - "TEST_TOOL=bundler RGV=.. BDV=master"
+      env: "TEST_TOOL=bundler RGV=.. BDV=master"
+    - stage: test
```

Comment:
This stage will test Rubygems against vendored bundler right?

Reply:
Yes. The regular test run for rubygems adds the vendored bundler to the LOAD_PATH, so the few specs that use bundler should use the vendored one. This change affects only bundler's specs, that run against the rubygems checkout instead of against rubygems master. This change is similar to #2695, but for ruby-head.